### PR TITLE
i18n(es): Update `add-view-transitions.mdx`, `mixed-content-data-collection-error.mdx`, `error-reference` & `content-collection-type-mismatch-error.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description (required)

`add-view-transitions.mdx` https://github.com/withastro/docs/commit/e788cb13c48ac5b08556123f1cffb603d2d8e1ca
`mixed-content-data-collection-error.mdx` https://github.com/withastro/docs/commit/db02f4d609f6b92f9ffee93e6276f69c102383a8
`error-reference` https://github.com/withastro/docs/commit/db02f4d609f6b92f9ffee93e6276f69c102383a8
`content-collection-type-mismatch-error.mdx` https://github.com/withastro/docs/commit/db02f4d609f6b92f9ffee93e6276f69c102383a8

### CODE REVIEW

File: src/content/docs/es/reference/error-reference.mdx [hunk 1]
```diff
@@ -63,6 +63,9 @@ La siguiente referencia es una lista completa de los errores que puedes encontra
 - [**FailedToLoadModuleSSR**](/es/reference/errors/failed-to-load-module-ssr/)<br/>Could not import file.
 - [**InvalidGlob**](/es/reference/errors/invalid-glob/)<br/>Invalid glob pattern.
 - [**FailedToFindPageMapSSR**](/es/reference/errors/failed-to-find-page-map-ssr/)<br/>Astro couldn't find the correct page to render
+- [**MissingLocale**](/en/reference/errors/missing-locale/)<br/>The provided locale does not exist.
```
Comment (Waxer59) [inline src/content/docs/es/reference/error-reference.mdx:66]:
```suggestion
- [**MissingLocale**](/es/reference/errors/missing-locale/)<br/>The provided locale does not exist.
```

File: src/content/docs/es/reference/error-reference.mdx [hunk 2]
```diff
@@ -63,6 +63,9 @@ La siguiente referencia es una lista completa de los errores que puedes encontra
...
+- [**CantRenderPage**](/en/reference/errors/cant-render-page/)<br/>Astro can't render the route.
```
Comment (Waxer59) [inline src/content/docs/es/reference/error-reference.mdx:67]:
```suggestion
- [**CantRenderPage**](/es/reference/errors/cant-render-page/)<br/>Astro can't render the route.
```

File: src/content/docs/es/reference/error-reference.mdx [hunk 3]
```diff
@@ -63,6 +63,9 @@ La siguiente referencia es una lista completa de los errores que puedes encontra
...
+- [**UnhandledRejection**](/en/reference/errors/unhandled-rejection/)<br/>Unhandled rejection
```
Comment (Waxer59) [inline src/content/docs/es/reference/error-reference.mdx:68]:
```suggestion
- [**UnhandledRejection**](/es/reference/errors/unhandled-rejection/)<br/>Unhandled rejection
```

File: src/content/docs/es/reference/errors/content-collection-type-mismatch-error.mdx [hunk 4]
```diff
@@ -4,7 +4,7 @@ i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
 
-> `COLLECTION_NAME` contiene entradas del tipo `ACTUAL_TYPE`, pero está configurado como una colección de `EXPECTED_TYPE`.
+> **ContentCollectionTypeMismatchError**: La COLECCIÓN contiene TIPO_ESPERADO elementos, pero está configurada como una colección de TIPO_ACTUAL.
```
Comment (dreyfus92) [inline src/content/docs/es/reference/errors/content-collection-type-mismatch-error.mdx:7]:
```suggestion
> **ContentCollectionTypeMismatchError**: La COLECCIÓN contiene el TIPO_ESPERADO de elementos, pero está configurada como una colección de TIPO_ACTUAL.
```

File: src/content/docs/es/reference/errors/mixed-content-data-collection-error.mdx [hunk 5]
```diff
@@ -4,7 +4,7 @@ i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
 
-> `COLLECTION_NAME` contiene una combinación de entradas de contenido y datos. Todas las entradas deben ser del mismo tipo.
+> **MixedContentDataCollectionError**: **NOMBRE_COLECCIÓN* contiene una combinación de entradas de contenido y datos. Todas las entradas deben ser del mismo tipo.
```
Comment (dreyfus92) [inline src/content/docs/es/reference/errors/mixed-content-data-collection-error.mdx:7]:
```suggestion
> **MixedContentDataCollectionError**: **NOMBRE_COLECCIÓN** contiene una combinación de entradas de contenido y datos. Todas las entradas deben ser del mismo tipo.
```

**Review by dreyfus92 — COMMENTED** [review]
Buen trabajo @Waxer59, te dejo algunas sugerencias 😁

**Comment by Waxer59** [comment]
Listo! gracias por la revisión 🤠!

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Pull request merged**